Tools/Scripts/libraries/webkitapipy/webkitapipy/sdkdb.py

```diff
@@ -316,15 +320,39 @@ def add_allowlist(self, allowlist: Path) -> bool:
 
     def _add_allowlist(self, config: AllowList, allowlist: Path):
         for entry in config.allowed_spi:
+            cond_id = None
+            if entry.requires:
+                # Convert a requirements list like ["A", "B", "!C"] into a
+                # graph data structure e.g. (A) -> (B) -> (!C). The head node
+                # is associated with each allowed declaration. The audit() query
+                # cross-references condition chains with active conditions
+                # added by add_defines().
+                #
+                # FIXME: No effort is made to reuse nodes in the graph between
+                # allowlist entries, so we store more than we have to.
```

Comment:
Can you file a bug to track this and link it?

Reply:
Yes, thanks! https://bugs.webkit.org/show_bug.cgi?id=295819